src/EffectiveCSharp.Analyzers/AnalyzerReleases.Unshipped.md

```
@@ -5,3 +5,4 @@
 
 Rule ID | Category | Severity | Notes
 --------|----------|----------|-------
+ECS1400 | Initialization | Info | MinimizeDuplicateInitializationLogicAnalyzer, [Documentation](https://github.com/rjmurillo/EffectiveCSharp.Analyzers/blob/4f481355aaf72846c9ff201c6ccd8551bf7874f5/docs/rules/ECS1400.md)
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)